extension.neon

```diff
@@ -46,14 +46,14 @@ parameters:
     noUnnecessaryCollectionCall: true
     noUnnecessaryCollectionCallOnly: []
     noUnnecessaryCollectionCallExcept: []
-    databaseMigrationsPath: null
+    databaseMigrationsPath: []
     checkModelProperties: false
 
 parametersSchema:
     noUnnecessaryCollectionCall: bool()
     noUnnecessaryCollectionCallOnly: listOf(string())
     noUnnecessaryCollectionCallExcept: listOf(string())
-    databaseMigrationsPath: schema(anyOf(string()), nullable())
+    databaseMigrationsPath: listOf(string())
```

Comment:
Is removing nullable intentional?

Reply:
Yes it was intentional. It defaults to an empty array so there is no need for null handeling anywhere in the code.